iac-extensions/spring-config/src/main/java/org/sonar/iac/springconfig/parser/properties/SpringConfigPropertiesParser.java

```diff
@@ -32,7 +32,7 @@ public class SpringConfigPropertiesParser implements TreeParser<Tree> {
   @Override
   public SpringConfig parse(String source, @Nullable InputFileContext inputFileContext) {
     var inputCode = CharStreams.fromString(source);
-    var propertiesLexer = new PropertiesLexer(inputCode);
+    var propertiesLexer = new PropertiesLexer(inputCode, true);
```

Comment:
Is the second parameter needed only to select the correct overload?

Reply:
Yes